readme.md

```diff
@@ -0,0 +1,153 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>README</b></h3>
+</div>
+
+<br>
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+
+Hello Microverse is the first miniproject from module 1. This project serves to familiarize students with Git, Github,gitflow and linters.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[key_feature_1]**
+- **[key_feature_2]**
+- **[key_feature_3]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Any modern browser.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+
+### Install
+
+Install this project with:
+
+
+### Usage
+
+To run the project, execute the following command:
+
+
+### Run tests
+
+To run tests, run the following command:
+
+
+
+### Deployment
+
+You can deploy this project using:
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 Andrew Mccollin
+
+- GitHub: [@githubhandle](https://github.com/Drew246)
+- Twitter: [@twitterhandle](n/a)

```

Comment:
- [ ] Good job so far for adding all sections as requested previously 👍 but while doing so there is something you didn't hit well  kindly consider adding a working link to your Twitter account to give a good user experience to the people visiting your readme

Reply:
- [ ] Kindly remove the **Twitter** account if you do not have any 🤗 We do not want to confuse our users ❤️

Reply:
good idea thanks. Can you please tell me what the conflict is? also is it possible that we can do a quick video chat ? I am falling behind and have to be done by tomorrow.

Reply:
@Drew246 Sure, I'm calling you.